test/TestExpectations.json

```diff
@@ -3647,6 +3647,12 @@
     "parameters": ["firefox", "headful", "webDriverBiDi"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[launcher.spec] Launcher specs Puppeteer Browser.disconnect should reject navigation when browser closes",
```

Comment:
FF does not support 2 simultaneous sessions.

Reply:
This is a CDP expectation.

Reply:
I think CDP-related expectations should not change.

Reply:
Right, my bad. I'll check it out

Reply:
Apparently it was an artifact. Removed.